Research/LocalMultiwaySystem/LocalMultiwaySystem.md

```diff
@@ -0,0 +1,825 @@
+In that case, we stop enumeration and declare these two edges are not purely spacelike.
+
+Now, if we only add matches to the index that are purely pairwise spacelike as described above, we would get similar evolution as is performed by the global `MultiwaySystem`.
+
+It is interesting now to consider the same example of two particles starting on two different branches that [caused lots of redundancy](#global-multiway-system) in the global `MultiwaySystem`.
+
+```wl
+In[] := WolframModel[{{1}, {1, 2}} -> {{1, 2}, {2}}, {{a1}, {a1, a2}, {a2, 
+    a3}, {a3, m1}, {b1}, {b1, b2}, {b2, m1}, {m1, m2}}, 
+  Infinity]["LayeredCausalGraph"]
+```
+
+<img src="Images/RedundantLocalMultiway.png" width="328">
+
+Instead of a mesh of redundant events that the global multiway system produced, we now only have two places where the events merge.
+To examine them more closely, lets "label" the particles:
+
+```wl
+In[] := labeledCausalGraph[
+ WolframModel[{{p, x, 1}, {1, 2}} -> {{1, 2}, {p, x, 2}}, {{p, a, 
+    a1}, {a1, a2}, {a2, a3}, {a3, m1}, {p, b, b1}, {b1, b2}, {b2, 
+    m1}, {m1, m2}}, Infinity], ImageSize -> 384]
+```
+
+<img src="Images/RedundantLocalMultiwayLabeledParticles.png" width="527">
+
+If we look closely at the events near the merge points, we can see that some "redundancy" remains, but it is no longer due to the spacelike separated events, but rather due to the "background" edges being rewritten during evolution.
+And as a result, if a particle follows another particle on the same "track", the edges it's going through are different (even though they involve the same vertices), hence the "duplication".
+
+Note, however, that this kind of evolution would not give us quantum physics, as the branches never merge or interfere with one another.
+
+A more interesting case would be to consider a system of rules where some are purely spacelike (i.e., only operate on pairwise spacelike sets of elements), and some are purely (or impurely) branchlike (i.e., they only operate on branchlike separated edges, but not on spacelike ones).
+
+Investigation of these rules and the study of possible quantum effects in the match-all local multiway system are the subjects for future research.
+
+# Phases and Interference in Multiway Systems
+
+<img src="Images/ComplexInterference.png" width="478">
+
+## Introduction
+
+The goal of the [Wolfram Physics Project](https://www.wolframphysics.org) is to find the fundamental law of physics, a simple rule that describes the entire Universe.
+That includes quantum mechanics.
+There were attempts made to describe quantum mechanics by [Stephen Wolfram](https://www.wolframphysics.org/technical-introduction/) and [Jonathan Gorard](https://www.wolframcloud.com/obj/wolframphysics/Documents/some-quantum-mechanical-properties-of-the-wolfram-model.pdf), however, a consistent and complete formulation has not yet emerged.
```

Comment:
In what respects is the proof of the validity of the von Neumann construction of a projective Hilbert space from a multiway evolution graph (whose partial order satisfies the axioms of an orthomodular lattice) not "complete and consistent"? Or are you claiming that the underling construction of Menger and Birkhoff's continuous geometry is incomplete and/or inconsistent?